source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.md

```diff
@@ -284,6 +284,14 @@ Endpoint. The pool has the following properties:
 - **Rate-limited:** A Pool MUST limit the number of [Connections](#connection) being
     [established](#establishing-a-connection-internal-implementation) concurrently via the **maxConnecting**
     [pool option](#connection-pool-options).
+- **Backpressure-enabled** - The pool MUST add the error labels `SystemOverloadedError` and `RetryableError` to network
+    errors or network timeouts it encounters during the connection establishment or the `hello` message. These labels
+    are used by the
+    [SDAM error handling](../server-discovery-and-monitoring/server-discovery-and-monitoring.md#error-handling-pseudocode)
+    to avoid clearing the pool. The pool MUST NOT add the backpressure error labels during an authentication step
+    after the `hello` message. For errors that the driver can distinguish as never occurring due to server overload,
+    such as DNS lookup failures, TLS related errors, or errors encountered establishing a connection to a socks5 proxy,
+    the driver MUST NOT clear the connection pool and MUST NOT mark the server Unknown.
```

Comment:
Since we want to continue clearing the pool and marking the server unknown for definite non-overload errors, shouldn't this say "MUST" rather than "MUST NOT"?